recipe2023/index.html

```diff
@@ -1,80 +1,103 @@
+      or even ground beef. But in our case, we'll make it vegetarian friendly!
+      </div>
+    </div>
+    <br />
+    <br />
+    <br />
+    <div>
+      <img
+        src="https://media.istockphoto.com/id/1372510923/photo/kulit-pangsit-or-dumpling-skin-usually-as-a-wrapper-with-minced-meat-in-it.jpg?s=612x612&w=0&k=20&c=h7CdGllDW19jdXwVaRPnNXyYWPYmOvNo_XM73tqtG3M="
+        alt="wonton wrapper"
+        width="600"
+        height="450"
+      />
+      <div class="row">
+        <div class="column">
+          <h3>Ingredients</h3>
+          <ul class="removebullets">
+            <li>8-10 Momos , only filled and shaped, not steamed</li>
+            <li>1 clove Garlic , finely chopped</li>
+            <li>1 inch Ginger , grated</li>
+            <li>1 or 2 Green Chillies , finely chopped</li>
+            <li>
+              2 to 3 Spring Onion (Bulb & Greens) , finely chopped with leaves
+            </li>
+            <li>1 teaspoon Light soy sauce</li>
+            <li>1/2 teaspoon Sesame (Gingelly) Oil</li>
+            <li>1 teaspoon Rice Vinegar</li>
+          </ul>
+        </div>
 
-  <img
-    src="https://media.istockphoto.com/id/531567099/photo/wonton-soup-chinese-food.jpg?s=612x612&w=0&k=20&c=uyGfCs1W7H1THG4kEKm2eeYBZs2cuRHMOTeZPhKe7pI="
-    alt="a bowl of wonton soup"
-    width="600"
-    height="450"
-  />
+      <div class="column">
+        <h3>How to make Wonton Soup</h3>
 
-  <div>
-    Delicious homemade Wonton Soup is the best way to satisfy any craving for
-    Chinese food without having to leave the house. Tender wontons filled in a
-    clear broth flavored with ginger, soy, sesame, and garlic. Big batches of
-    wontons can be made ahead of time and frozen for a quick and yummy meal in
-    minutes.
-  </div>
-  <h3>What is Wonton Soup?</h3>
-  <div>
-    Wonton soup is a standard of Chinese cuisine and made from seasoned chicken
-    broth with filled wontons. Wontons are an Asian equivalent of ravioli or
-    tortellini when they are folded around a spiced meat mixture. This recipe
-    usually calls for ground pork as the foundation for the filling because it
-    has great flavor and a meaty texture. Other meats or proteins could be
-    substituted, such as shrimp, ground chicken or turkey, or even ground beef.
-    But in our case, we'll make it vegetarian friendly!
-  </div>
+        <ul class="removebullets2">
+          <li>1.Place broth and seasoning in a pot and bring to a boil.</li>
+          <li>2.Remove the ginger and garlic and reduce heat.</li>
+          <li>3.Gently simmer the wontons in the broth until done.</li>
+          <li>
+            4.Serve in bowls or cups garnished with chopped green onions or
+            chives. Remember to treat the wontons gently to prevent them from
+            tearing! If you’d like to add veggies and shrimp to turn this into a
+            Wor Wonton Soup, these wontons work well for that too!
+          </li>
+        </ul>
```

Comment:
```suggestion
        <ol class="removebullets2">
          <li>Place broth and seasoning in a pot and bring to a boil.</li>
          <li>Remove the ginger and garlic and reduce heat.</li>
          <li>Gently simmer the wontons in the broth until done.</li>
          <li>
            Serve in bowls or cups garnished with chopped green onions or
            chives. Remember to treat the wontons gently to prevent them from
            tearing! If you’d like to add veggies and shrimp to turn this into a
            Wor Wonton Soup, these wontons work well for that too!
          </li>
        </ol>
```

If you use an `<ol>` instead of `<ul>`, you'll automatically get the numbering for you